src/PharoLauncher-CLI/PhLImageCopyCliCommand.class.st

```diff
@@ -49,11 +49,13 @@ PhLImageCopyCliCommand class >> launcherCmdPositionals [
 
 { #category : #'command execution' }
 PhLImageCopyCliCommand >> executeWithImage: anImage [
-	[ self imageRepository copyImage: anImage to: self targetName ]
+	[[ self imageRepository copyImage: anImage to: self targetName ]
 		on: PhLNameNotAvailableError
 		do: [ ^ self errorStream
 				nextPutAll: self targetName,'Name already used';
-				newLine ]
+				newLine ]]
+		on: MessageNotUnderstood 
```

Comment:
does not look related to the feature + we should not rely on MessageNotUnderstood for business code